src/stackTools/stackScroll.js

```diff
@@ -45,12 +45,17 @@ function mouseWheelCallback (e) {
   const config = stackScroll.getConfiguration();
 
   let loop = false;
+  let allowSkipping = true;
 
   if (config && config.loop) {
     loop = config.loop;
   }
 
-  scroll(eventData.element, images, loop);
+  if (config && config.allowSkipping !== undefined) {
```

Comment:
You should be able to merge this check and default with the above check. For example:

_Previous:_
```
if (config && config.loop) {
  loop = config.loop;
}
```

_Becomes:_
```
if (config) {
  loop = config.loop !== undefined ? config.loop : false;
  allowSkipping = config.allowSkipping !== undefined ? || true;
}

Reply:
Can do.